src/detect-pangram.js

```diff
@@ -15,7 +15,16 @@
  * @return {boolean}
  */
 function detectPangram(phrase) {
-  // write code here
+  const sentence = phrase.toLowerCase();
```

Comment:
уточни в каком регистре
```suggestion
  const phraseLowerCase = phrase.toLowerCase();
```